src/main/java/uk/gov/hmcts/juror/api/moj/service/DeferralResponseServiceImpl.java

```diff
@@ -103,12 +103,21 @@ private void declineDeferralForJurorPool(BureauJwtPayload payload, DeferralReque
 
```

Comment:
Need a migration to identify / fix jurors in prod that are already effected by this issue